lib/Parser/RegexParser.cpp

```diff
@@ -2948,7 +2949,7 @@ namespace UnifiedRegex
     {
         Assert(program->source == 0);
 
-        program->source = _u("");
+        program->source = const_cast<Char*>(_u(""));
```

Comment:
That was the first thing I tried but quite a bit of code assumes its not const so went with the cheaper fix of maintaining the current assumption